node/executor/src/lib.rs

```diff
@@ -241,7 +247,7 @@ mod tests {
 			None,
 		).0.unwrap();
 		let r = ApplyResult::decode(&mut &v.as_encoded()[..]).unwrap();
-		assert_eq!(r, Err(ApplyError::CantPay));
+		assert_eq!(r, Err(InvalidTransactionValidity::Payment.into()));
```

Comment:
This terminology is pretty opaque.

By "invalid transaction validity", do you mean that the *validity* of this transaction is invalid? Or the transaction itself? (In which case, it should read "invalid transaction".)

In any case, it replaces `ApplyError::CantPay`, which is